sys/fuchsia/pager.txt

```diff
@@ -0,0 +1,23 @@
+# Copyright 2022 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+# See https://cs.opensource.google/fuchsia/fuchsia/+/main:zircon/vdso/pager.fidl
+
+include <zircon/syscalls.h>
+include <zircon/types.h>
+
+resource zx_pager[zx_handle]
+
+zx_pager_create(options flags[pager_create_opt], out ptr[out, zx_pager])
+zx_pager_create_vmo(pager zx_pager, options flags[pager_create_vmo_opt], port zx_port, key proc[1000, 16], size int64, out ptr[out, zx_vmo])
+zx_pager_detach_vmo(pager zx_pager, vmo zx_vmo)
+zx_pager_supply_pages(pager zx_pager, pager_vmo zx_vmo, offset int64, length int64, aux_vmo zx_vmo, aux_offset int64)
+zx_pager_op_range(pager zx_pager, op flags[pager_op_range_opt], pager_vmo zx_handle, offset int64, length int64, data int64)
+
+# TODO: syscalls marked `@next` are not yet included in the VDSO and need an additional header included if we want to fuzz them.
```

Comment:
I *think* (?) that we should therefore comment these out until we are ready to fuzz them, because Syzkaller will crash or otherwise generate a useless error when it tries to call these functions and the VDSO support for them isn't there.

Reply:
The `(disabled)` attribute on the definition has this effect, I think: "the call will not be used in fuzzing; useful to temporary disable some calls or prohibit particular argument combinations" (from https://github.com/google/syzkaller/blob/master/docs/syscall_descriptions_syntax.md), and it seems like the advantage to that (vs. commenting out) is that it's easier to grep for definitions that need some fixing up before being enabled.

Reply:
Ohhh, you are right and I am silly.